### ISSUE TITLE
[React OverflowMenu]: All content must reside within a HTML5 Sectioning element, WAI-ARIA landmark or labelled region role.

### ISSUE THREAD

**Issue by kkomprakash** [carbon-design-system/carbon#5337]
## [React OverflowMenu]: All content must reside within a HTML5 Sectioning element, WAI-ARIA landmark or labelled region role.

-->

> Automated testing tool and ruleset

Violation:
All content must reside within a HTML5 Sectioning element, WAI-ARIA landmark or labelled region role.
Orphan content found that does not reside within a WAI-ARIA landmark or labelled region role.

> Assistive technology used to verify
DAP extension
## Detailed description

Able to see this AVT issue in https://react.carbondesignsystem.com/?path=/story/overflowmenu--basic

> What version of the Carbon Design System are you using?
 "carbon-components": "^10.0.0",
    "carbon-components-react": "^7.0.0",

> What did you expect to happen?

No AVT issue should be there

> What happened instead?
AVT issue is be there

> What WCAG 2.1 checkpoint does the issue violate?
https://aat.w3ibm.mybluemix.net/token/a1f74e74-a651-4035-ab2f-16b524d25c84/f4811aea-af41-43c1-b82f-61d78aec22c5/archives/2019SeptDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1157.html

## Steps to reproduce the issue

1. Open overflow menu in chrome browser
2. Inspect with [DAP](https://chrome.google.com/webstore/detail/dynamic-assessment-plugin/fnapgcgphlfhecijolobjodbbnjjpdga) extension

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/38567891/74390008-d4ed6080-4db4-11ea-83ca-6349b06d62ca.png)

**Comment by joshblack** [comment]
Hi there! 👋 I believe this violation is due to our usage of Storybook. When used in an application with landmarks, this will not be a violation.

Going to close this due to the above, let me know if this is not the case!

**Comment by kkomprakash** [comment]
Hi @joshblack , I see that when overflow menu is opened, the floating menu is rendered at the end of `body` tag which makes it fall outside `section` or landmarks. That's why we are seeing this AVT issue. 

Codesandbox link - https://codesandbox.io/s/overflowmenuissue-96xgi

Screenshot with issue:
![image](https://user-images.githubusercontent.com/38567891/74775665-06e64300-524b-11ea-8b68-53c21f31a30d.png)

Is there any solution apart from adding `role` attribute to `body` tag?

**Comment by vpicone** [comment]
The button should have an aria-controls that refers to the new region

**Comment by vpicone** [comment]
@kkomprakash We walk up the tree looking for the nearest element with `[data-floating-menu-container]`. `document.body` is  used as a fallback. Try adding that attribute to your section.

https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/OverflowMenu/OverflowMenu.js#L450
